lib/utilities/ConcordiaConstants.dart

```diff
@@ -34,6 +34,241 @@ const String FG_BUILDING_INITIAL = "FG";
+  },
+  {
+    'Building': 'GS Building',
+    'code': 'GS',
+    'xcoords': [45.496639, 45.496507, 45.496473, 45.496592],
+    'ycoords': [-73.581134, -73.581239, -73.581118, -73.581031],
+  },
+  {
+    'Building': 'K Annex',
+    'code': 'K',
+    'xcoords': [45.497834, 45.497794, 45.497705, 45.497737],
+    'ycoords': [-73.579535, -73.579470, -73.579558, -73.579630],
+  },
+  {
+    'Building': 'LD Building',
+    'code': 'LD',
+    'xcoords': [45.496525, 45.496561, 45.496856, 45.496856],
+    'ycoords': [-73.577388, -73.577459, -73.577164, -73.577164],
+  },
+  {
+    'Building': 'Learning Square Building',
+    'code': 'LS',
+    'xcoords': [45.495952, 45.496111, 45.496177, 45.496156, 45.496188, 45.496052],
+    'ycoords': [-73.579633, -73.579479, -73.579643, -73.579669, -73.579741, -73.579866],
+  },
+  {
+    'Building': 'M Annex',
+    'code': 'M',
+    'xcoords': [45.497291, 45.497323, 45.497420, 45.497390],
+    'ycoords': [-73.579778, -73.579845, -73.579751, -73.579681],
+  },
+  {
+    'Building': 'MI Annex',
+    'code': 'MI',
+    'xcoords': [45.497777, 45.497809, 45.497645, 45.497621, 45.497694, 45.497682],
+    'ycoords': [-73.579171, -73.579230, -73.579398, -73.579352, -73.579282, -73.579259],
+  },
+  {
+    'Building': 'MU Annex',
+    'code': 'MU',
+    'xcoords': [45.497750, 45.497919, 45.497957, 45.497783],
+    'ycoords': [-73.579628, -73.579464, -73.579541, -73.579698],
+  },
+  {
+    'Building': 'P Annex',
+    'code': 'P',
+    'xcoords': [45.496584, 45.496675, 45.496720, 45.496626],
+    'ycoords': [-73.579181, -73.579082, -73.579167, -73.579266],
+  },
+  {
+    'Building': 'PR Annex',
+    'code': 'PR',
+    'xcoords': [45.496961, 45.497060, 45.497045, 45.496947],
+    'ycoords': [-73.579973, -73.579867, -73.579839, -73.579941],
+  },
+  {
+    'Building': 'Q Annex',
+    'code': 'Q',
+    'xcoords': [45.496554, 45.496582, 45.496670, 45.496643],
+    'ycoords': [-73.579121, -73.579173, -73.579082, -73.579030],
+  },
+  {
+    'Building': 'R Annex',
+    'code': 'R',
+    'xcoords': [45.496713, 45.496802, 45.496834, 45.496747],
+    'ycoords': [-73.579424, -73.579330, -73.579400, -73.579483],
+  },
+  {
+    'Building': 'RR Annex',
+    'code': 'RR',
+    'xcoords': [45.496649, 45.496791, 45.496756, 45.496613],
+    'ycoords': [-73.579476, -73.579333, -73.579259, -73.579402],
+  },
+  {
+    'Building': 'S Annex',
+    'code': 'S',
+    'xcoords': [45.497366, 45.497487, 45.497459, 45.497436, 45.497424, 45.497329],
+    'ycoords': [-73.579931, -73.579815, -73.579756, -73.579776, -73.579762, -73.579860],
+  },
+  {
+    'Building': 'Samuel Bronfman Building',
+    'code': 'SB',
+    'xcoords': [45.496533, 45.496541, 45.496621, 45.496672],
+    'ycoords': [-73.586209, -73.586041, -73.585972, -73.586086],
+  },
+  {
+    'Building': 'T Annex',
+    'code': 'T',
+    'xcoords': [45.496665, 45.496742, 45.496711, 45.496631],
+    'ycoords': [-73.579336, -73.579256, -73.579189, -73.579269],
+  },
+  {
```

Comment:
Duplicates for TD?

Reply:
Updated the file to remove that duplicate.